modules/mobile/app/sidekiq/mobile/v0/pre_cache_claims_and_appeals_job.rb

```diff
@@ -30,8 +31,49 @@ def perform(uuid)
 
       private
 
-      def claims_proxy(user)
```

Comment:
A lot of these methods are copied from the controller. following the rule of 3, I decided not to create a new layer to abstract it out